src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -561,8 +571,16 @@ namespace dd
 
         try
           {
-            _inputIndex = _engine->getBindingIndex("data");
-            _outputIndex0 = _engine->getBindingIndex(out_blob.c_str());
+            _inputIndex = 0;
+            if (out_blob == "last")
+              _outputIndex0 = _engine->getNbBindings() - 1;
+            else
+              _outputIndex0 = _engine->getBindingIndex(out_blob.c_str());
+            _dims = _engine->getBindingDimensions(_outputIndex0);
+            this->_logger->info("detected output dimensions: [{}, {} {} {}]",
+                                _dims.d[0], _dims.d[1],
```

Comment:
what happens if nbDims < 2? segfault?

Reply:
Yeah the logger will crash. I can catch that error, rational being 2 dims being the absolute minimal since batch size x uni dimensional tensor.